registry/registry_db.go

```diff
@@ -73,6 +109,23 @@ https://www.alexedwards.net/blog/organising-database-access
 */
 var db *sql.DB
 
+func (st ServerType) String() string {
+	return string(st)
+}
+
+func (rs RegistrationStatus) String() string {
+	switch rs {
+	case 1:
```

Comment:
Have the case be the various names (`Pending`, `Approved`, `Denied`) instead of the integer values (1, 2, 3).